documentation/asciidoc/computers/configuration/pin-configuration.adoc

```diff
@@ -24,7 +22,7 @@ Note that it may take a few seconds to get from stage 1 to stage 4. During that
 * Delay the peripheral's startup until stage 4/5 has been reached
 * Add an appropriate pull-up/-down resistor
 
-=== Providing a Custom Device Tree Blob
+=== Provide a custom Device Tree blob
```

Comment:
There's a bit a few lines above here where it says `dtblob.bin` when it should be `dt-blob.bin`.